js/tawk_to.widget.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+jQuery(document).ready(function() {
+  var tawktoSettings = Drupal.settings.tawkto || {};
+
+  window.Tawk_LoadStart = new Date();
+  window.Tawk_API = window.Tawk_API || {};
+
+  if (tawktoSettings.user && tawktoSettings.enableVisitorRecognition) {
+    window.Tawk_API.visitor = {
+      name : tawktoSettings.user.name,
+      email : tawktoSettings.user.email
+    };
+  }
+
+  var curpage = window.location.href;
+  var hash = window.location.hash;
+  window.Tawk_API.onLoad = function() {
+    if (curpage.indexOf('admin/') !== -1 || hash.indexOf('overlay')!== -1) {
```

Comment:
Spacing issue

Reply:
I think you may have missed that the line before is declaring a function. That's why this line has another indentation.